File: SalesforceTraining/Day7/FirstJava/src/T11.java
// reverse decimal

import java.util.Scanner;

public class T11 {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter number");
        float number = scanner.nextFloat();
        String reverse = "";
        String str = Float.toString(number);
        // System.out.println(str);
        String[] array = str.split("\\.");
        int i = str.indexOf('.');

        //new
        StringBuilder sb = new StringBuilder(array[0]);
        sb.reverse();

        reverse = array[1] + sb.toString();


        // reverse = array[1] + array[0];
        // reverse = reverse.substring(0,i) + "." + reverse.substring(i,reverse.length());
        StringBuilder addDot = new StringBuilder(reverse);
        addDot.insert(i, ".");
        reverse = addDot.toString();
        System.out.println(reverse);
        scanner.close();

    }
}

File: day3/T11.js
const getOne = (...args) =>{
    let sum = 0
    for(i of args){
        sum += i
    }
    while(sum.toString().length != 1){
        sum = sum.toString().split("")
        let mul = 1
        for(i of sum){
            mul *= Number(i)
        }
        sum = mul
    }
    console.log(sum)

}
getOne(16,28)